src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,440 @@
+            onClick={() => {
+              openMenu()
+            }}
+            sx={{
+              overflow: "hidden",
+              position: "absolute",
+              textOverflow: "ellipsis",
+              whiteSpace: "nowrap",
+              width: inputRef.current?.getClientRects()[0].width,
+            }}
+          >
+            {renderSelectedItem
+              ? renderSelectedItem(selectedOption)
+              : optionToString(selectedOption)}
+          </div>
+        )}
+        {buttons && buttons.length > 0 && selectedOption && !isOpen && (
+          <div
+            sx={{
+              display: "flex",
+              gap: 1,
+              whiteSpace: "nowrap",
+            }}
+          >
+            {buttons.map((button, index) => (
+              <Button
+                key={`autocompleteButton-${index}`}
+                {...button}
+                sx={{ paddingY: 0 }}
+              >
+                {button.children}
+              </Button>
+            ))}
+          </div>
+        )}
+      </div>
+      {helperText && (
+        <div
+          sx={{
+            ...{
+              default: {
+                color: "text.40",
+                variant: "text.body2",
+              },
+              loading: {},
+              success: {
+                px: 3,
+                pb: 2,
+                color: "success.0",
+                variant: "text.body1",
+              },
+              error: {
+                px: 3,
+                pb: 2,
+                color: "primary.0",
+                variant: "text.body1",
+              },
+            }[state],
+          }}
+        >
+          {helperText}
+        </div>
+      )}
+      <div
+        {...getMenuProps()}
+        sx={{
+          background: "#fff",
+          border: "1px solid #E8E8E9",
+          borderRadius: 4,
+          boxShadow: "0px 1px 12px rgba(0, 0, 0, 0.16)",
+          marginTop: 3,
+          minWidth: containerRef.current?.getClientRects()[0].width,
+          outline: "0",
+          overflowX: "hidden",
+          overflowY: "auto",
+          padding: 4,
+          paddingBottom: totalCount ? 0 : 4,
+          position: "absolute",
+          transition: "opacity .1s ease",
+          top:
+            window.scrollY +
+            (containerRef.current?.getClientRects()[0].bottom || 0),
+          visibility: isOpen ? "visible" : "hidden",
+          zIndex: 1000,
+          "& > div": {
+            cursor: "pointer",
+          },
+        }}
+      >
+        {isOpen &&
+          options.map((item, index, array) => (
+            <div {...getItemProps({ item, index })} key={index}>
+              {renderOption(
+                highlightedIndex,
+                {
+                  backgroundColor:
+                    highlightedIndex === index
+                      ? "secondary.alpha.95"
+                      : undefined,
+                  borderRadius: 2,
+                  height: 60,
```

Comment:
Isn't it better to just set the padding to 2 in the default styling, and let the height adjust itself to the children?
From the design it seems the height is 60px only because of the content within that example and the 8px padding, not because it should always be 60px